# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/nx-X11/installinopt.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN nx-X11/config/cf/site.def nx-X11-new/config/cf/site.def
--- nx-X11/config/cf/site.def	2002-02-27 01:51:12.000000000 +0100
+++ nx-X11-new/config/cf/site.def	2004-03-25 13:12:17.926673776 +0100
@@ -72,23 +72,25 @@
 #ifdef AfterVendorCF
 
 #ifndef ProjectRoot
-#define ProjectRoot /usr/X11R6
+#define ProjectRoot /opt/nx
 #endif
 
 /*
  * On some platforms, some things may be installed outside of ProjectRoot
  * by default.  To disable this, uncomment the following line.
  *
-#define NothingOutsideProjectRoot YES
  */
+#define NothingOutsideProjectRoot YES
+#define FontconfigFontsConfDir ProjectRoot/etc/fonts
+#define LinkGLToUsrInclude NO
 
 /*
  * Set EtcX11Directory if you want config file links installed under
  * ProjectRoot rather than in /etc/X11.  See also HasVarDirectory,
  * UseEtcX11 and UseSeparateConfDir.
  *
-#define EtcX11Directory ProjectRoot/etc
  */
+#define EtcX11Directory ProjectRoot/etc
 
 
 /* Only set HasXdmAuth to YES if you have a Wraphelp.c file. */
